functions.php

```diff
@@ -84,10 +84,23 @@ function hello_elementor_setup() {
 				add_theme_support( 'wc-product-gallery-slider' );
 			}
 		}
+
+		hello_maybe_update_theme_version_in_db();
```

Comment:
Move to the top of the function, and run only if `is_admin()` (Always avoid run `update` queries from the front)

Reply:
Done @matipojo